medusa/service/grpc/medusa_pb2_grpc.py

```diff
@@ -74,125 +79,153 @@ def GetBackups(self, request, context):
+
 
 def add_MedusaServicer_to_server(servicer, server):
     rpc_method_handlers = {
-        'Backup': grpc.unary_unary_rpc_method_handler(
-            servicer.Backup,
-            request_deserializer=medusa__pb2.BackupRequest.FromString,
-            response_serializer=medusa__pb2.BackupResponse.SerializeToString,
-        ),
-        'AsyncBackup': grpc.unary_unary_rpc_method_handler(
-            servicer.AsyncBackup,
-            request_deserializer=medusa__pb2.BackupRequest.FromString,
-            response_serializer=medusa__pb2.BackupResponse.SerializeToString,
-        ),
-        'BackupStatus': grpc.unary_unary_rpc_method_handler(
-            servicer.BackupStatus,
-            request_deserializer=medusa__pb2.BackupStatusRequest.FromString,
-            response_serializer=medusa__pb2.BackupStatusResponse.SerializeToString,
-        ),
-        'DeleteBackup': grpc.unary_unary_rpc_method_handler(
-            servicer.DeleteBackup,
-            request_deserializer=medusa__pb2.DeleteBackupRequest.FromString,
-            response_serializer=medusa__pb2.DeleteBackupResponse.SerializeToString,
-        ),
-        'GetBackups': grpc.unary_unary_rpc_method_handler(
-            servicer.GetBackups,
-            request_deserializer=medusa__pb2.GetBackupsRequest.FromString,
-            response_serializer=medusa__pb2.GetBackupsResponse.SerializeToString,
-        ),
+            'Backup': grpc.unary_unary_rpc_method_handler(
+                    servicer.Backup,
+                    request_deserializer=medusa__pb2.BackupRequest.FromString,
+                    response_serializer=medusa__pb2.BackupResponse.SerializeToString,
+            ),
+            'AsyncBackup': grpc.unary_unary_rpc_method_handler(
+                    servicer.AsyncBackup,
+                    request_deserializer=medusa__pb2.BackupRequest.FromString,
+                    response_serializer=medusa__pb2.BackupResponse.SerializeToString,
+            ),
+            'BackupStatus': grpc.unary_unary_rpc_method_handler(
+                    servicer.BackupStatus,
+                    request_deserializer=medusa__pb2.BackupStatusRequest.FromString,
+                    response_serializer=medusa__pb2.BackupStatusResponse.SerializeToString,
+            ),
+            'DeleteBackup': grpc.unary_unary_rpc_method_handler(
+                    servicer.DeleteBackup,
+                    request_deserializer=medusa__pb2.DeleteBackupRequest.FromString,
+                    response_serializer=medusa__pb2.DeleteBackupResponse.SerializeToString,
+            ),
+            'GetBackups': grpc.unary_unary_rpc_method_handler(
+                    servicer.GetBackups,
+                    request_deserializer=medusa__pb2.GetBackupsRequest.FromString,
+                    response_serializer=medusa__pb2.GetBackupsResponse.SerializeToString,
+            ),
+            'PurgeBackups': grpc.unary_unary_rpc_method_handler(
```

Comment:
Do we have an idea how long this takes to return a response on average?  Would this be something we do `async` and have the controller check back for status?

If so, we would have a corresponding `PurgeStatus` similar to the `AsyncBackup` model.

Reply:
Purge should be much faster than backups, so I thought it wasn't necessary for the initial implementation.
Let's get a feel for how that goes for now. Also there is some refactoring planned to make the orchestration code common between MedusaTasks and MedusaBackupJobs which might be a good place to do this.